charts/capsule/README.md

```diff
@@ -99,6 +99,7 @@ Here the values you can override:
 |-----|------|---------|-------------|
 | affinity | object | `{}` | Set affinity rules for the Capsule pod |
 | certManager.generateCertificates | bool | `false` | Specifies whether capsule webhooks certificates should be generated using cert-manager |
+| certManager.generateCertificates | list | `[]` |  Specify additional SANS to add to the certificate |
```

Comment:
```suggestion
| certManager.additionalSANS | list | `[]` |  Specify additional SAN to add to the certificate |
```

Reply:
I guess the Chart lint has spotted this:

```diff
@@ -98,8 +98,8 @@ Here the values you can override:
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` | Set affinity rules for the Capsule pod |
+| certManager.additionalSANS | list | `[]` | Specify additional SANS to add to the certificate |
 | certManager.generateCertificates | bool | `false` | Specifies whether capsule webhooks certificates should be generated using cert-manager |
-| certManager.generateCertificates | list | `[]` |  Specify additional SANS to add to the certificate |
 | customAnnotations | object | `{}` | Additional annotations which will be added to all resources created by Capsule helm chart |
 | customLabels | object | `{}` | Additional labels which will be added to all resources created by Capsule helm chart |
 | imagePullSecrets | list | `[]` | Configuration for `imagePullSecrets` so that you can use a private images registry. |
```